app/resources/start.html.template

```diff
@@ -17,19 +17,21 @@
 
   <div class="main">
     <div class="row">
-      <div class="action-outer start">
-        <a href="{{seek_url}}" tabindex="1">
-          <div class="action-inner" onclick="location.href={{seek_url}};" tabindex="1" style="cursor:pointer;">
+      {# Following onclick is for making the whole button clickable purpose. #}
```

Comment:
Sorry, I messed something during pushing the altered code.
Would you please see the[ latest code](https://github.com/google/personfinder/pull/364/files)?